java/arcs/core/host/ProdHost.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.host
+
+import com.google.auto.service.AutoService
```

Comment:
It's temporarily until we introduce our own annotation processor similar to sting. We basically need an APT that will collect all annotations and generate a master injector for us, purely based on build time classpath scanning.  AutoService is runtime JAR scanning of META-INF/

Reply:
The annotations classes are harmless and won't break JS or WASM since the annotation processor only generates an extra text file in the output. The ServiceLoader code however is JVM dependent and shouldn't be in arcs.core